src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -51,6 +51,8 @@ public static void loadModElements() {
 		ModElementType.ENCHANTMENT = register(
 				new ModElementType<>("enchantment", 'm', BaseType.ENCHANTMENT, RecipeType.NONE, EnchantmentGUI::new,
 						Enchantment.class));
+		ModElementType.FEATURE = register(
+				new ModElementType<>("feature", null, BaseType.FEATURE, RecipeType.NONE, FeatureGUI::new, Feature.class));
```

Comment:
As the feature mod element will be surely used more than the fluid ME, I think the Feature ME should have the `u` and the Fluid ME should not have a shortcut. Features have more usage than fluids.

Reply:
After the food element is removed, we could use `f` for features

Reply:
This probably got forgotten as now letter f should be free